libs/detectors/detector.py

```diff
@@ -11,7 +11,10 @@ def __init__(self, config, source):
         self.config = config
         self.device = self.config.get_section_dict("Detector")["Device"]
         self.resolution = tuple([int(i) for i in self.config.get_section_dict("App")["Resolution"].split(",")])
+        # TODO: Iterator in the below line should be replaced with:
```

Comment:
Why this? Can we solve it?

Reply:
It was a reminder for myself, we should discuss this personally.